src/main/java/seedu/address/model/types/event/UniqueEventList.java

```diff
@@ -28,6 +31,50 @@ public class UniqueEventList implements Iterable<Event> {
     private final ObservableList<Event> internalUnmodifiableList =
             FXCollections.unmodifiableObservableList(internalList);
 
+    /**
+     * Periodically sorts the events especially when an event becomes completed.
+     */
+    //    public UniqueEventList() {
+    //        Timeline eventReSortTimeline = DateTimeUtil.createTimeline(this::sortEvents,
+    //                javafx.util.Duration.minutes(1));
+    //        eventReSortTimeline.play();
+    //    }
+
+    private void sortEvents() {
+        internalList.sort((event1, event2) -> {
+            long remainingTime1 = getEventTimeRemaining(event1);
```

Comment:
Nitpicking, but maybe can rename these variables to something clearer, eg. e1RemainingTime and e2RemainingTime